components/Auth.js

```diff
@@ -28,21 +30,23 @@ export default function Auth({ handleLogin }) {
 
   return (
     <View style={styles.container}>
-      <Spacer size={2} />
-      <Title style={styles.title}>{UI_STRINGS.heading}</Title>
-      <Subheading style={styles.subheading}>{UI_STRINGS.subheading}</Subheading>
-      <Spacer size={2} />
-      <View>
-        <Button
-          style={styles.button}
-          accessibilityLabel="login"
-          mode="contained"
-          icon="google"
-          onPress={handleLogin}
-        >
-          {UI_STRINGS.button}
-        </Button>
-      </View>
+      <Spacer size={12} />
+      <Headline color={theme.colors.surface}>{UI_STRINGS.heading}</Headline>
+      <Spacer size={4} />
+      <BodyText style={styles.subheading}>{UI_STRINGS.subheading}</BodyText>
+      <Spacer size={12} />
+      <Button
+        style={styles.button}
+        accessibilityLabel="login"
+        mode="contained"
+        icon="google"
```

Comment:
I believe the icon can also be an image. see the paper docs. using the string icon name will give you [this](https://materialdesignicons.com/icon/google), whose color can't be customized as in the designs

Reply:
Good point. I tried this earlier but I was missing the wrapper `Image` component. Here's the commit with the updated UI https://github.com/nearform/optic-expo/pull/21/commits/9477af4da284bbc74add4267e54bc65ec1a97330